Safety/out-of-hours-support.rst

```diff
@@ -0,0 +1,88 @@
+.. An error will alert you of identical labels during the build process.
+.. _`Emergency Call List`: https://start.1password.com/open/i?a=EJGC5QLBFNAJ3ODCNVC2I3HIPM&v=mqlr5qdkpipoyamzhty5kg365y&i=tu4v5aa4tvhbm4e3rc5siiwada&h=lsstit.1password.com
+.. _`download`: https://1password.com/downloads
+.. _`IT support`: https://jira.lsstcorp.org/projects/IHS/issues/IHS-6753?filter=allopenissues
+.. _`#auxtel-operations`: https://lsstc.slack.com/archives/CLQ63T89L
+.. _`#auxtel-virtual-control-room`: https://lsstc.slack.com/archives/C01K4M6R4AH
+.. _`#rubin-observing`: https://lsstc.slack.com/archives/C04HULH5HHD
+
+####################
+Out of hours support
+####################
+
+The Out of Hours support call list is managed in a 1Password vault named "Emergency Call List".
+All observing specialists are automatically enrolled in the vault once they have a 1Password account.
+You must submit a JIRA ticket to `IT support`_ to be enrolled in the Rubin 1Password system. 
+Once enrolled, you can `download`_ a desktop application or browser extension to use 1Password and access the information the `Emergency Call List`_ vault. 
+
+.. note::
+    If you volunteer to be on the Emergency Call List, you will see the phone numbers provided by every other volunteer.
+    Your number will also be accessible to everyone with permissions to the vault.
+    If you volunteer, please provide your availability to the best of your knowledge, including holdiays and weekends.
+
+.. _out-of-hours-support-procedure:
+
+Procedure
+^^^^^^^^^
+
+- If a fault happens during the night, or some new behavior appears in the system, observers and summit staff should attempt to troubleshoot to the extent of their knowledge.
+
+- Ask for help on Slack:
+    - `#auxtel-operations`_
+    - `#auxtel-virtual-control-room`_
+    - `#rubin-observing`_
+
+- If no response is received from Slack, and summit staff have run out of ideas or need assistance, activate the Emergency Call List.
+
+    - Under comments, the volunteer's area of expertise should be listed. 
+    - If they don't answer, search for the next available person with similar expertise.
+    - If no one answers, and the problem cannot be solved or worked-around, close and stow the telescope(s) by 2am, if this is possible. 
+
+.. note::
+    If any equipment is sensed to be in danger, exhuast the call list until you receive an answer, and report the problem. 
+    Leave the equipment in the safest configuration possible, if you have the knowledge to do so. 
+
+.. _out-of-hours-support-calling-instructions:
+
+Calling instructions
+^^^^^^^^^^^^^^^^^^^^
+
+- Refer to the :ref:`instructions <summit-contact-numbers-using-the-control-room-phones>` on how to use the summit control rooms and dial an outside line. 
+
+- The Rubin Control room number appears on a caller ID as +1 520-318-8800.
```

Comment:
This is if the person's cell is based in the US. Presumably the number is different if you have a Chilean phone?

Reply:
Need to test. I can call myself when I am at summit.

Reply:
👍